Infrastructure.Services/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <configSections>
+        <sectionGroup name="applicationSettings" type="System.Configuration.ApplicationSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
+            <section name="Infrastructure.Services.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+        </sectionGroup>
+    </configSections>
+    <applicationSettings>
+        <Infrastructure.Services.Properties.Settings>
+            <setting name="KLEOnlineUrl" serializeAs="String">
+                <value>http://api.kle-online.dk/resources/kle</value>
```

Comment:
@strongmindsmhs Jeg er usikker på om det her rent faktisk vil virke. Det er jo web.config, som er den aktive config for den kørende app og deri ligger der settings, som opdateres på deployment tidspunktet.

Reply:
@strongmindsmhs Det lader til også at virke på integration, så det er nok *.settings filen der leverer standardværdien... det er fint